demo/index.html

```diff
@@ -26,6 +26,11 @@
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@aurodesignsystem/design-tokens@latest/dist/tokens/CSSCustomProperties.css">
     <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/@aurodesignsystem/webcorestylesheets@latest/dist/demoWrapper.css" />
     <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/@aurodesignsystem/webcorestylesheets@latest/dist/elementDemoStyles.css" />
+    <style>
+      .exampleWrapper--ondark {
```

Comment:
I implemented this into auro-docsite for my local beta. This will get us past the theme demo for now. Once you approve this PR I will make a final edit to remove this custom style locally.